CHANGES

```diff
@@ -158,7 +154,16 @@ Notations
   Note that this might cause incompatibilities if you have, e.g., list_scope
   and vector_scope both open with vector_scope on top, and expect `++` to
   refer to `app`.
-  Solution: wrap `_ ++ _` in `(_ ++ _)%list` (or whichever scope you want). 
+  Solution: wrap `_ ++ _` in `(_ ++ _)%list` (or whichever scope you want).
```

Comment:
Why does GitHub think this line has changed? :confused:

Reply:
I auto-deleted a trailing whitespace.